pkgs/development/compilers/gcc/11/default.nix

```diff
@@ -27,6 +27,15 @@
 , cloog # unused; just for compat with gcc4, as we override the parameter on some places
 , buildPackages
 , libxcrypt
+, nukeReferences
+, callPackage
+, enableGdbPlugin ? enablePlugin
+# perform the bootstrap in Nix and configure with `--disable-internal-bootstrap`
```

Comment:
What is `--disable-internal-bootstrap`? Did you mean gcc's `--disable-bootstrap`? I find "external bootstrap" as a nem very confusing.

Reply:
Fixed: 7231abcb610dcb91c7e67f2728c552683e793194

Reply:
> I find "external bootstrap" as a nem very confusing.

I'm open to suggestions.

I don't want to call it "disable bootstrap" because we're not disabling it.  It is reimplemented (including the checksumStage!) in Nix.  So there needs to be an adjective in there.  But I'm not particularly attached to "external"; if there's a better adjective we should use that instead.

Reply:
`disableGccAutotoolsBootstrap` would be more accurate.  But it is pretty clumsy.

Reply:
How about "Nix-driven bootstrap"?

Reply:
How about `enableNixBootstrap`?

Reply:
I don't really have any particular preference, just throwing ideas out there.

Reply:
That is a `gcc` derivation. If we consider it as a `gcc` package don't you just pass `--disable-bootstrap`? Or you build `gcc` derivation twice under the hood using this flag?

If it's just a glorified `--disable-bootstrap` I would call it that. If it implements the two builds it should state it explicitly at least in the comment.

Reply:
> If it's just a glorified `--disable-bootstrap`

No.

To make things completely clear: I have factored out the two-line change which is "just a glorified `--disable-bootstrap`" and submitted that as a separate PR: https://github.com/NixOS/nixpkgs/pull/216136

Reply:
> How about "Nix-driven bootstrap"?

I like this a lot, especially because I can use the same terminology to explain other things, like why `crate2nix` is so much better than `buildRustPackage`.

I've changed the name from "external bootstrap" to "nix driven bootstrap": 90b4b2817c5ba925af3bfd21139a8e493c0be347